src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -339,6 +339,21 @@ type StellarCoreCfg =
         if self.forceOldStyleLeaderElection then
             t.Add("FORCE_OLD_STYLE_LEADER_ELECTION", true) |> ignore
 
+        if self.network.missionContext.runForMaxTps then
+            if not self.network.missionContext.enableInMemoryBuckets then
+                t.Add("BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT", 0) |> ignore
+
+            if not self.network.missionContext.enableParallelApply then
+                t.Add("EXPERIMENTAL_PARALLEL_LEDGER_APPLY", true) |> ignore
+
```

Comment:
Can you please add background signature verification (if it's not already enabled) here too?

Reply:
good catch, fixed!